climate_categories/tests/test_iso3.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for the 'ISO3' categorization."""
+import pytest
+
+import climate_categories
+
+
+def test_categories():
+    assert climate_categories.ISO3["DEU"].title == "Germany"
+    assert len(climate_categories.ISO3) > 150
+
+
+def test_eu():
+    assert len(climate_categories.ISO3["EU-12"].children[0]) == 12
+    assert len(climate_categories.ISO3["EU-15"].children[0]) == 15
+    assert len(climate_categories.ISO3["EU-25"].children[0]) == 25
+    assert len(climate_categories.ISO3["EU-27_2007"].children[0]) == 27
+    assert len(climate_categories.ISO3["EU-28"].children[0]) == 28
+    assert len(climate_categories.ISO3["EU-27_2020"].children[0]) == 27
+    with pytest.raises(KeyError):
+        climate_categories.ISO3["EU27"]
+
+    assert climate_categories.ISO3["EU"] == climate_categories.ISO3["EU_2020"]
+
+
+def test_unfccc():
+    assert len(climate_categories.ISO3["Annex-I"].children[0]) == 43
+    assert len(climate_categories.ISO3["Non-Annex-I"].children[0]) == 155
+    # leaf children excludes EU, because EU is not a leaf
+    assert len(climate_categories.ISO3["UNFCCC"].leaf_children[0]) == 197
```

Comment:
In the NDC tool, we use 196 countries. What is the extra one here?

Reply:
The country which is a party to the UNFCCC but not in the NDC tool is :drum: :
```python
('VAT',
 {'title': 'Holy See (Vatican City State)',
  'alternative_codes': ['VA', '336']})
 ```

Reply:
They've been a party to the UNFCCC [since September last year already](https://unfccc.int/node/61239), try to keep up :wink:

Reply:
The IAM groups like R5, RC10 are useful but have to be defined on a per IAM basis. I spend quite some time gathering this info for the SSP downscaling, but probably it has changed now as IAMs probably are now more detailed. So I think we should add them only when used.

An regarding changing the country terminology name from ISO3 to whatever. I prefer if it stays the same as that's probably hardcoded in a lot of places as I almost never use other country terminologies. But of course it's doable to change it.